src/routes/routes.js

```diff
@@ -49,10 +53,11 @@ const Routes = () => {
 
   return (
     <ConnectedRouter history={history}>
-      <Suspense fallback={<Loader />}>
+      <Suspense fallback={<div />}>
         <ErrorBoundary>
           <AppHeader />
           <div className={styles.root}>
+            <ToastContainer autoClose={3000} hideProgressBar />
```

Comment:
I think it's better to set this props from store.

Reply:
done